.idea/dictionaries/harry.xml

```diff
@@ -0,0 +1,8 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="harry">
```

Comment:
Why harry ? :) Is this what Android Studio generated by default?

Reply:
Android studio uses the username of the user saving the word, and that is my username on my primary development machine.

Reply:
IntelliJ/AS namespaces dictionaries to the user who saved the word to avoid merge conflicts. Each user will generate a new dictionary for their username, however the saved words are used by all users regardless of whose dictionary the word is saved to. I'd prefer it be saved under a name, but I have yet to find a way to get AS to save it elsewhere.